scripts/add_peers.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import subprocess
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser(prog="Add known peers to IPFS config")
+    parser.add_argument("--file", "-f", type=str, default="known_peers.yaml")
+
+    args = parser.parse_args()
+
+    with open(args.file, "r") as fp:
+        peers = yaml.safe_load(fp)
+
+    peer_config = json.dumps(
+        [{"ID": p["ID"], "Addrs": p.get("Addrs", [])} for p in peers]
+    )
+    subprocess.run(["ipfs", "config", "--json", "Peering.Peers", peer_config])
```

Comment:
In case there are already some peers in the config, those would be overwritten by this step (I guess). I'd recommend by default to keep current entries. E.g. by first reading previous entries and then merging them with the new ones.
I'd probably even merge the list of Addrs if there's currently an ID which is to be added.
Maybe overwriting the entire list should be an option to the script though.